app/src/main/java/org/eu/exodus_privacy/exodusprivacy/CheckAppActivity.java

```diff
@@ -106,13 +106,8 @@ public void onSuccess(Application application) {
                 dialogBuilder.setMessage(getString(R.string.app_not_analyzed));
                 //noinspection RedundantSuppression
                 dialogBuilder.setPositiveButton(R.string.submit, (dialog, id) -> {
-                    Uri uri;
+                    Uri uri = Uri.parse("https://exodus.phm.education.gouv.fr/analysis/submit/#" + app_id);
```

Comment:
```suggestion
                    Uri uri = Uri.parse("https://reports.exodus-privacy.eu.org/analysis/submit/#" + app_id);
```

I think that's the wrong one!

Reply:
Fixed